process/templates/PROJECT_NAME_sandbox_stage.md

```diff
@@ -15,11 +19,14 @@ When contributing an existing Project to the OpenSSF, the contribution must unde
 ### Project References
 The project should provide a list of existing resources with links to the repository, and if available, website, a roadmap, contributing guide, demos and walkthroughs, and any other material to showcase the existing breadth, maturity, and direction of the project.
 
-| Reference          | URL |
-|--------------------|-----|
-| Repo               |     |
-| Website            |     |
-| Contributing guide |     |
-| Roadmap            |     |
-| Demos              |     |
-| Other              |     |
+| Reference           | URL |
+|---------------------|-----|
+| Repo                |     |
+| Meeting Agenda      |     |
+| OSSF Calendar Entry |     |
```

Comment:
```suggestion
```

These tables are hard! I can see why it'd be useful to have meeting agendas, if available, but after spending several hours reading these doc I *just now* realized that anything beyond the repository is optional. We don't reference an agenda / calendar meeting though, so I think it makes sense to remove it since it isn't clear if it's optional.

Reply:
I agree. My initial intention with this stuff was to keep it as short as possible. My feeling was that a link to the main TI's page/repo should provide most of what we need. If it doesn't then we can then ask for additional info when we do the review.